jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -464,13 +464,12 @@
       pick-result))
   (:pick-object-with-movable-region
     (arm movable-region &key (n-trial 1) (n-trial-same-pos 1)
-         (do-stop-grasp nil) (grasp-style :suction))
+         (do-stop-grasp nil) (grasp-style :suction) (object-index 0))
     "Return value: :grasp-succeeded, :grasp-failed or :ik-failed"
     (send *ri* :stop-grasp arm :pinch)
     (send *ri* :calib-proximity-threshold arm)
-    (let (pick-result avs object-index obj-pos obj-cube pinch-yaw suction-yaw near-walls)
+    (let (pick-result avs obj-pos obj-cube pinch-yaw suction-yaw near-walls)
       ;; pick largest bounding box
```

Comment:
Fix this comment like:
`pick largest bounding box when object-index is 0`

Reply:
thx, i add the comment.